src/gemini/gemini-config.ts

```diff
@@ -31,7 +33,7 @@ export const askGeminiText = async (question: string) => {
 
 export const askGeminiVision = async (
   question: string,
-  images: imageProps[]
+  images: string[]
 ) => {
   const model = geminiAI.getGenerativeModel({ model: "gemini-pro-vision" });
 
```

Comment:
Can also delete the comment on line 42! 

// will need to be adjusted based on image location